src/components/interactive-map.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import autobind from '../utils/autobind';
 
 import StaticMap from './static-map';
-import MapState, {MAPBOX_MAX_PITCH, MAPBOX_MAX_ZOOM} from '../utils/map-state';
+import {MAPBOX_MAX_PITCH, MAPBOX_MAX_ZOOM} from '../utils/map-state';
```

Comment:
Q: should we make a namespace so the user doesn't have to import a "laundry list" of these 

`MAPBOX_LIMITS.MAX_PITCH` etc?